Gemfile

```diff
@@ -4,7 +4,7 @@ gem 'cocoapods'
 
 # A reference to Maze Runner is only needed for running tests locally and if committed it must be
 # portable for CI, e.g. a specific release.  However, leaving it commented out would mean quicker CI.
-gem 'bugsnag-maze-runner', git: 'https://github.com/bugsnag/maze-runner', tag: 'v3.7.0'
+gem 'bugsnag-maze-runner', git: 'https://github.com/bugsnag/maze-runner', tag: 'v4.6.0'
```

Comment:
I don't think we want to change this yet as it will break local running of the Expo and Node tests, which have not been moved to MR v4 yet and do not have their own Gemfile(.lock)s.  

For now I think I'd simply revert this file and `Gemfile.lock`, but once the Expo and Node scenarios have been updated to v4 we can lose them completely.